src/processor/queue-consumer.service.ts

```diff
@@ -98,8 +100,8 @@ export class QueueConsumerService extends WorkerHost implements OnApplicationBoo
         const delay = blocksRemaining * SECONDS_PER_BLOCK * MILLISECONDS_PER_SECOND;
         this.logger.debug(`Adding delay to job ${job.id} for ${delay}ms`);
         const {key: delayJobId, data: delayJobData} = createGraphUpdateJob(job.data.dsnpId, job.data.providerId, job.data.processTransitiveUpdates);
-        await this.graphUpdateQueue.add(delayJobId, delayJobData, {delay});
-        return;
+        await this.graphUpdateQueue.removeRepeatableByKey(delayJobId);
```

Comment:
I might be mistaken but I don't think the jobs that you are adding are repeatable jobs. If not you should be able to just use `remove`.